webpack.dev.js

```diff
@@ -0,0 +1,12 @@
+const { merge } = require('webpack-merge');
```

Comment:
This file intended?

Reply:
no, it's been removed from master, let me delete